deps/third/xmake.lua

```diff
@@ -1,11 +1 @@
-includes("zycore")
-includes("zydis")
-includes("polyhook_2")
-includes("glaze")
-includes("imgui")
-includes("glad")
-includes("raw_pdb")
-
-add_requires("zycore v1.5.0", { debug = is_mode_debug(), configs = {runtimes = get_mode_runtimes()} })
-add_requires("zydis v4.1.0", { debug = is_mode_debug(), configs = {runtimes = get_mode_runtimes()} })
-add_requires("polyhook_2", { debug = is_mode_debug(), configs = {runtimes = get_mode_runtimes()} })
\ No newline at end of file
+includes("glad")
```

Comment:
Is there an argument to package-ize all our third party deps? Glad has a package def that we can rip off (or install from xrepo)

https://github.com/xmake-io/xmake-repo/blob/0cc9a8d6ce18f77e92b2155f9f87d1509c5b530b/packages/g/glad/xmake.lua#L4

Does our glad do anything special?

Reply:
I prefer to keep everything a target when possible, glad is also something that can change based on the configuration we generate it in.